routes/writing.js

```diff
@@ -42,8 +45,31 @@ router.get("/viewAllBlogs", function (req, res) {
   });
 });
 
-router.post("/post", function (req, res) {
-  var reqBlog = req.body.blog;
+router.post("/post",[
+
+  body('blog.name')
+    .trim().not().isEmpty(),
+   body('nodes.*.content')
+      .trim().not().isEmpty()
+
+
+], function (req, res) {
+
+  const errors=validationResult(req);
+  console.log(errors);
+  if (!errors.isEmpty()) {
+
+      res.send(500);
+    console.log(errors);
+    //  console.log(req.body.nodes[0].content);
+
+  }
+else{
```

Comment:
Use try and catch blocks for error handling btw. This helps in sending error responses for all sorts of errors, instead of sending individual response for particular errors.